### DISCUSSION TITLE
"Uninstall" function not work propertly

### DISCUSSION THREAD

**Discussion by vampywiz17** [LingmoOS/Community#15] [Bugs Report/缺陷提交] [answered]
right click "uninstall" not work properly , need to use "apt remove/autoremove" command in terminal, if you want to uninstall it correctly.

**Comment by elysia-best** [comment]
Thanks for you report. The old uninstaller is known to be broken and is being rewritten. I'll transfer this issue to the correct place.

**Comment by elysia-best** [comment, marked as answer]
Moved to https://github.com/LingmoOS/lingmo-launcher/issues/4
